src/pages/de/core-concepts/framework-components.md

```diff
@@ -0,0 +1,234 @@
+import vue from '@astrojs/vue';
+import solid from '@astrojs/solid-js';
+import lit from '@astrojs/lit';
+import alpine from '@astrojs/alpinejs';
+
+export default defineConfig({
+	integrations: [react(), preact(), svelte(), vue(), solid(), lit(), alpine()],
+});
+```
+
+⚙️ Schaue den [Integrationsleitfaden](/en/guides/integrations-guide/) für mehr Details an und erfahre mehr über das Installieren und Konfigurieren von Astro-Integrationen.
+
+⚙️ Möchtest du ein Beispiel für ein Framework deiner Wahl sehen? Besuche [astro.new](https://astro.new/) und wähle eine Framework-Vorlage.
+
+## Framework-Komponenten nutzen
+
+Nutze deine JavaScript Framework-Komponenten in deinen Astro-Seiten, -Layouts und -Komponenten genauso wie deine Astro-Komponenten. Alle deine Komponenten können zusammen in `/src/components` organisiert werden, oder auf eine andere Weise, die dir gefällt.
+
+Um eine Framework-Komponente zu nutzen, importiere sie relativ in deine Astro-Komponente. Anschließend kannst du die Komponente neben anderen Komponenten, HTML-Elementen und JSX-ähnlichen Ausdrücken im Komponenten-Template verwenden.
+
+```astro title="src/pages/static-components.astro" ins={2,7}
+---
+import MyReactComponent from '../components/MyReactComponent.jsx';
+---
+<html>
+  <body>
+    <h1>Nutze React-Komponenten direkt in Astro!</h1>
+    <MyReactComponent />
+  </body>
+</html>
+```
+
+Standardmäßig werden die Framework-Komponenten als statisches HTML gerendert. Dies ist hilfreich für Template-Komponenten, die nicht interaktiv sind. Dadurch wird verhindert, dass unnötiges JavaScript an den Client gesendet wird.
+
+## Interaktive Komponenten hydratisieren
+
+Eine Framework-Komponenten kann kann aktiviert (hydratisiert) werden, indem die `client:*`-Direktiven verwendet werden. Das ist ein Komponenten-Attribut, welches definiert wie die Komponente **gerendert** und **hydriert** werden soll.
+
+Die [Client-Direktive](/en/reference/directives-reference/#client-directives) beschreibt ob die Komponente zur Build-Zeit gerendet werden soll und wann das JavaScript der Komponente client-seitig durch den Browser geladen werden soll. 
+
+Die meisten Direktiven rendern die Komponenten auf dem Server zur Build-Zeit. Das Komponenten-JS wird entsprechend der angewendeten Direktive an den Client gesendet. Die Komponente wird hydratisiert, wenn ihr JS geladen wurde.
+
+```astro title="src/pages/interactive-components.astro" /client:\S+/
+---
+// Beispiele: Framework-Komponenten im Browser hydratisiert.
+import InteractiveButton from '../components/InteractiveButton.jsx';
+import InteractiveCounter from '../components/InteractiveCounter.jsx';
+---
+<!-- Das JS dieser Komponente wird geladen, wenn die Seite lädt-->
+<InteractiveButton client:load />
+
+<!-- Das JS dieser Komponente wird nicht an den Client übertragen solange der User 
+ nicht auf der Seite herunterscrollt und die Komponente auf der Seite sichtbar ist -->
+<InteractiveCounter client:visible />
+```
+
+:::Achtung
```

Comment:
Not commenting on the details of this because it's no longer present in the English version, but I'd like to point out that the names of directives starting with `:::` (like `:::note`, `:::caution` etc.) must not be translated, or they won't work.